great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -351,12 +352,17 @@ def _estimate_metric_value_range(
         lower_bound: Optional[float] = truncate_values.get("lower_bound")
         upper_bound: Optional[float] = truncate_values.get("upper_bound")
 
-        round_decimals: int = self._get_round_decimals_using_heuristics(
-            metric_values=metric_values,
-            domain=domain,
-            variables=variables,
-            parameters=parameters,
-        )
+        round_decimals: int
+
+        if integer_semantic_domain_type(domain=domain):
```

Comment:
`integer_semantic_domain_type` is the helper method that was introduced as part of ths PR.

Reply:
This is a **much** simpler way of doing the work for implementing the `interpolation_method`